README.md

```diff
@@ -28,7 +28,7 @@ Presuming you are running some Linux with a `sudo` capable user, the following s
 
   ```bash
   git clone https://github.com/dune-community/Dockerfiles.git docker-for-daily-dune
-  ln -s docker-for-daily-dune/docker_* .
+  ln -s docker-for-daily-dune/docker_run.sh .
```

Comment:
Was this a problem on the mac? Since there is also a file `docker_exec.sh` that we need you would then need to add a second line
```bash
ln -s docker-for-daily-dune/docker_exec.sh .
```